modules/Emails/javascript/Email.js

```diff
@@ -291,7 +291,7 @@ function multiFiles( list_target){
         imgElement.setAttribute("width","16");
 
         var new_row_button_embed = document.createElement("img");
-        new_row_button_embed.setAttribute("src", "index.php?entryPoint=getImage&themeName="+SUGAR.themes.theme_name+"&imageName=attachment.gif");
+        new_row_button_embed.setAttribute("src", "index.php?entryPoint=getImage&themeName="+SUGAR.themes.theme_name+"&imageName=attachment.svg");
```

Comment:
@lewisje1991 , This functionality is work for all themes. There is backward compatibility previous themes